index.js

```diff
@@ -4,6 +4,14 @@ var mktemp = require('mktemp')
 var rimraf = require('rimraf')
 var underscoreString = require('underscore.string')
 
+exports.buildTmpPath = buildTmpPath
+function buildTmpPath(obj, prop, className) {
+  if (className == null) className = obj.constructor && obj.constructor.name
+
+  var tmpDirName = prettyTmpDirName(className, prop)
+  return obj[prop] = path.join(findBaseDir(), tmpDirName)
```

Comment:
The `obj[prop] =` assignment wasn't in the function this was extracted from, so we're technically assigning twice when you call `makeOrRemake` now. I'm not sure if this actually breaks anything.

Other than that I'm fine with this PR.

Out of curiosity, are we still using this for anything post Broccoli 1.0?